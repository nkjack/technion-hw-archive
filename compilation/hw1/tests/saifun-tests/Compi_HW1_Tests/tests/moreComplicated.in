/*

sads

	*/
/**/
HELLO COMPI PEOPLE !iMpOrtant
-0
+0
-0x1aF
1337
-0x1qF
-0xq
+
1.1
0x
15cm
+13px
rgb(1,1,1)
rgb(15,20,30)
rgb(015,0,-514)
rgb(1 ,	
	2,   03 )
rgb(1,,)

