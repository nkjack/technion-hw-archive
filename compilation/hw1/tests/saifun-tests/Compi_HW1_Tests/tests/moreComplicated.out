5 COMMENT 5
6 COMMENT 1
7 NAME HELLO
7 NAME COMPI
7 NAME PEOPLE
7 IMPORTANT !iMpOrtant
8 NUMBER -0
9 NUMBER +0
10 NUMBER -0x1aF
11 NUMBER 1337
12 NUMBER -0x1
12 NAME qF
13 NUMBER -0
13 NAME xq
14 COMB +
15 NUMBER 1
15 DOT .
15 NUMBER 1
16 UNIT 0x
17 UNIT 15cm
18 NUMBER +13
18 NAME px
19 RGB rgb(1,1,1)
20 RGB rgb(15,20,30)
21 RGB rgb(015,0,-514)
23 RGB rgb(1 ,	
	2,   03 )
Error in rgb parameters
